fb.php

```diff
@@ -0,0 +1,137 @@
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="area.php"><i class="fa fa-clone"></i>Area</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="vol.php"><i class="fa fa-prescription-bottle"></i>Volume and Capacity</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="mass.php"><i class="fa-solid fa-scale-unbalanced-flip"></i>Mass and Weight</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="speed.php"><i class="fa fa-tachometer-average"></i>Speed</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="temp.php"><i class="fa fa-temperature-high"></i>Temprature</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="cur.php"><i class="fa fa-thin fa-wallet"></i>Currency</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+        <div class="feedback">
+            <div class="container">
+                <div class="contact">
+                    <form action="#" method="POST">
+                        <div class="mb-3">
+                            <label for="exampleInputEmail1" class="form-label">Email address</label>
+                            <input type="email" class="form-control" name="email" id="email1" aria-describedby="emailHelp">
+                            <div id="emailHelp" class="form-text">We'll never share your email with anyone else.</div>
+                        </div>
+                        <div class="mb-3">
+                            <label for="exampleInputEmail1" class="form-label">Subject</label>
+                            <input type="text" class="form-control" name="sub" id="exampleInputEmail1" aria-describedby="emailHelp">
+                        </div>
+                        <div class="mb-3">
+                            <label for="exampleInputEmail1" class="form-label">Enter Your Feedback</label>
+                            <textarea class="form-control" name="msg" id="exampleInputEmail1" aria-describedby="emailHelp"></textarea>
+                        </div>
+                        <div class="text-center"> <button type="submit" name="submit" class="btn btn-primary">Submit</button></div>
+                    </form>
+                    <?php
+                    if (isset($_POST["submit"])) {
+                        $to = "premarch567@gmail.com";
+                        $sub = $_POST["sub"];
+                        $msg = $_POST["msg"];
+                        $from = $_POST["email"];
+                        $header = "From : $from";
+                        $a = 'mt-5';
+                        if (mail($to, $sub, $msg, $header)) {
```

Comment:
Instead of this you can use  ternary operator here for this if condition.

$counter = mail($to, $sub, $msg, $header) ? 1 : 0;
like this.